### PR TITLE
Update Ubuntu docs

### PR DESCRIPTION
**Pull request by mamedin**
This PR updates the Ubuntu sections for AM18, removing the obsolete Ubuntu
14.04 /Trusty) sections and adding new sections for Ubuntu 18.04 (Bionic).

Connects to https://github.com/archivematica/Issues/issues/203

Connects to  https://github.com/archivematica/Issues/issues/272

### CODE REVIEW

File: admin-manual/installation-setup/installation/install-ansible.rst [hunk 1]
```diff
@@ -65,17 +65,6 @@ Instructions
...
-   .. _ubuntu-14.04:
-
-   Ubuntu 14.04 (Trusty):
-
-   .. code:: bash
-
-      cd deploy-pub/playbooks/archivematica-trusty
-      ansible-galaxy install -f -p roles/ -r requirements.yml
-
-   .. _ubuntu-16.04:
```
Comment (sevein) [inline admin-manual/installation-setup/installation/install-ansible.rst]: Is it ok to remove this?

File: admin-manual/installation-setup/installation/install-ubuntu.rst [hunk 2]
```diff
@@ -417,6 +280,6 @@ the configuration of your new server.
 .. _`known issue`: https://github.com/artefactual/archivematica-storage-service/issues/312
 .. _`Sword API`: https://wiki.archivematica.org/Sword_API
 .. _`known issue with pip`: https://bugs.launchpad.net/ubuntu/+source/python-pip/+bug/1658844
-.. _`Dashboard install README`: https://github.com/artefactual/archivematica/blob/stable/1.7.x/src/dashboard/install/README.md
-.. _`MCPClient install README`: https://github.com/artefactual/archivematica/blob/stable/1.7.x/src/MCPClient/install/README.md
+.. _`Dashboard install README`: https://github.com/artefactual/archivematica/blob/stable/1.8.x/src/dashboard/install/README.md
+.. _`MCPClient install README`: https://github.com/artefactual/archivematica/blob/stable/1.8.x/src/MCPClient/install/README.md
```
Comment (sevein) [inline admin-manual/installation-setup/installation/install-ubuntu.rst:285]: `MCPServer install README` still points to `stable/1.7.x`.

**Review by sevein — COMMENTED** [review]
A couple of minor observations.

**Comment by sallain** [comment]
Just a reminder to update the table that appears on the [installation landing page](https://www.archivematica.org/en/docs/archivematica-1.8/admin-manual/installation-setup/installation/installation/#installation) by editing the CSV file located at `admin-manual/installation-setup/installation/_csv`

File: admin-manual/installation-setup/installation/install-ubuntu.rst (hunk 2 above)
Comment (mamedin) [reply admin-manual/installation-setup/installation/install-ubuntu.rst:285]: Thanks Jesus. Good catch! Just fixed.

File: admin-manual/installation-setup/installation/install-ansible.rst (hunk 1 above)
Comment (mamedin) [reply admin-manual/installation-setup/installation/install-ansible.rst]: Thanks, I deleted this line by mistake. Just restored.

**Comment by mamedin** [comment]
Thanks @sallain, I updated the table.

After including your suggestions it's ready for CR again. :)

**Review by sallain — APPROVED** [review]
LGTM!

File: admin-manual/installation-setup/installation/install-ubuntu.rst [hunk 3]
```diff
@@ -6,45 +6,56 @@ Manually Installing Archivematica on Ubuntu
...
 
+   Ubuntu 16.04 (Xenial):
+
    .. code:: bash
 
-      sudo wget -O - https://packages.archivematica.org/1.7.x/key.asc  | sudo apt-key add -
-      sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.7.x/ubuntu xenial main" >> /etc/apt/sources.list'
-      sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.7.x/ubuntu-externals xenial main" >> /etc/apt/sources.list'
+      sudo wget -O - https://packages.archivematica.org/1.8.x/key.asc  | sudo apt-key add -
+      sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.8.x/ubuntu xenial main" >> /etc/apt/sources.list'
```
Comment (scollazo) [inline admin-manual/installation-setup/installation/install-ubuntu.rst:31]: In the past, we have pointed the url for development packages here, and change it to the production one once it has been released.

**Review by scollazo — CHANGES_REQUESTED** [review]

**Pull request merged**